Docs about building the firmware seem to be incorrect

<https://cfw.ftcommunity.de/ftcommunity-TXT/en/advanced/building/building.html>

> Change to this directory:
> cd ftcommunity-TXT
> 
> Now prepare the build process:
> make fischertechnik_TXT_defconfig
> (this should take only minute or so)

If I follow the instructions, I get the following error
```
make fischertechnik_TXT_defconfig
make: *** Keine Regel, um „fischertechnik_TXT_defconfig“ zu erstellen.  Schluss.
```

--- a/docs/de/advanced/building/building.md
+++ b/docs/de/advanced/building/building.md
@@ -38,16 +38,12 @@ Dies erstellt ein neues Verzeichnis mit dem Namen ftcommunity-TXT im aktuellen V
 Wechsle in dieses Verzeichnis:
 ``````````cd ftcommunity-TXT``````````
 
-Bereite jetzt den Build-Prozess vor:
-``````````make fischertechnik_TXT_defconfig``````````  
-(Das sollte nur eine Minute dauern) 
-
-Starte nun den eigentlichen Build-Prozess
+Starte nun den Build-Prozess
 ``````````make  ``````````
 (Beim ersten Lauf dauert dies mehrere Stunden, abhängig von der CPU- und der Download-Geschwindigkeit)
 
 Achtung: Der Internetzugang ist während des Buildprozesses permanent erforderlich, da weitere Dateien (mehrere Gigabyte) heruntergeladen werden müssen.
-Wenn du einen Offline-Build erstellen möchten und nur alle zuvor über ``make fischertechnik_TXT_defconfig`` ausgewählten Quellen herunterladen möchten, verwende
+Wenn du einen Offline-Build erstellen möchtest, verwende
 ``make source  ``  
 vor dem eigentlichen ``make  ``  
 

--- a/docs/de/advanced/building/building.md
+++ b/docs/de/advanced/building/building.md
@@ -45,7 +45,7 @@ Starte nun den Build-Prozess
 Achtung: Der Internetzugang ist während des Buildprozesses permanent erforderlich, da weitere Dateien (mehrere Gigabyte) heruntergeladen werden müssen.
 Wenn du einen Offline-Build erstellen möchtest, verwende
 ``make source  ``  
-vor dem eigentlichen ``make  ``  
+im Ordner ``buildroot`` vor dem eigentlichen ``make`` im Hauptordner.  
 
 ### Nebenbemerkung - Den Build-Prozess beschleunigen
 Um den Prozess zu beschleunigen, kannst du den Build-Prozess starten, indem du den eigentlichen Build-Prozess mit

--- a/docs/en/advanced/building/building.md
+++ b/docs/en/advanced/building/building.md
@@ -45,7 +45,7 @@ Now start the actual build process
 Attention: Internet access is required permanently during the build process as further files (several gigabytes) have to be downloaded.  
 If you intend to do an offline build and just want to download all sources then use  
 ``make source  ``  
-before the actual ``make  ``  
+in directory ``buildroot`` before the actual ``make`` in the main directory.
 
 ### Side remark – Speeding up the build process
 To speed up the process, you can run the build process by starting the actual build process with  

--- a/docs/en/advanced/building/building.md
+++ b/docs/en/advanced/building/building.md
@@ -38,16 +38,12 @@ This creates a new directory named ftcommunity-TXT in the current (=home) direct
 Change to this directory:  
 ``````````cd ftcommunity-TXT``````````
 
-Now prepare the build process:  
-``````````make fischertechnik_TXT_defconfig``````````  
-(this should take only minute or so) 
-
 Now start the actual build process  
 ``````````make  ``````````
 (on first run this will take several hours, depending on CPU and download speed)
 
 Attention: Internet access is required permanently during the build process as further files (several gigabytes) have to be downloaded.  
-If you intend to do an offline build and just want to download all sources previously selected via ``make fischertechnik_TXT_defconfig`` then use  
+If you intend to do an offline build and just want to download all sources then use  
 ``make source  ``  
 before the actual ``make  ``  
 

